packages/fxa-payments-server/src/components/AppLayout/index.tsx

```diff
@@ -17,6 +17,7 @@ export const AppLayout = ({ children }: AppLayoutProps) => {
         data-testid="stage"
         className="fade-in-forward"
         style={{ opacity: 1 }}
+        lang={navigator.language}
```

Comment:
I added in this logic, in `PlanDetails/index.scss`, to set styling based on browser language.

`    
    .label:lang(en) {
      margin-top: 5px;
    }
    .total-price:lang(en) {
      font-size: 22px;
    }
`

Currently the lang is hardcoded to `en` in `index.html`. Ideally we'd use something like `react-helmet` to set the language but that seemed like a bigger change. So in the mean time I thought I'd just add it to `<AppLayout>`

Reply:
Thanks for the explanation.  I don't really like having to set language specific styles; but since this is what Zachery asked for, it's to spec I suppose.

Do you know if this also works if the language is 'en-CA' or 'en-GB'?

Reply:
Yup I also don't really like it. Could get hairy to maintain. I'm happy to push this back to Zachary and discuss further.

Yup it works for 'en-*'. Looking at the spec, it looks like :lang() does [implicit wildcard matching](https://drafts.csswg.org/selectors/#the-lang-pseudo). I also tested it manually quickly.

Reply:
> 
> 
> Yup I also don't really like it. Could get hairy to maintain. I'm happy to push this back to Zachary and discuss further.
> 

Yeah, let's.  (I like the smaller font size anyway. 😛)

> Yup it works for 'en-*'. Looking at the spec, it looks like :lang() does [implicit wildcard matching](https://drafts.csswg.org/selectors/#the-lang-pseudo). I also tested it manually quickly.

Nice.  At least we got that going for us.

Reply:
@chenba I heard back from Zachery and he was happy to continue with the smaller text for all languages. I've updated the CSS and removed the `lang` attribute from `<AppLayout>`.